ui/k-toolbar/main.kytos

```diff
@@ -12,6 +12,7 @@
                     placeholder="First endpoint" icon="arrow-right"
                     :candidates="dpids" 
```

Comment:
@rmotitsuki I wonder if we should exclude the local port `4294967294` for instance,  on a dpid 1, `00:00:00:00:00:00:00:01:4294967294`.

Reply:
Results are coming from REST service. It would be better to create another service without local ports. I will create an issue for this.